Ginger/GingerCoreNET/ValueExpression/ValueExpessionGeneralFunctions.cs

```diff
@@ -50,6 +51,17 @@ public string GetGUID()
             return Guid.NewGuid().ToString();
         }
 
+        [ValueExpressionFunctionAttribute]
+        [ValueExpressionFunctionDescription("Get Clipboard Text")]
+        [ValueExpressionFunctionExpression("{Function Fun=GetClipboardText()}")]
+        [ValueExpressionFunctionCategory("Data")]
+        [ValueExpressionFunctionSubCategory("Functions")]
+        public string GetClipboardText()
+        {
+            return WorkSpace.Instance.OSHelper.GetClipboardText();
```

Comment:
we said to push it later once we'll find solution for it, no?

Reply:
I only added the basic implementation, So I will comment this code so I can easily come back to implement it later